### PR TITLE
Added a static setString method for externals

### PR DESCRIPTION
**Pull request by stevechae**

### CODE REVIEW

File: src/android/nl/xservices/plugins/LaunchMyApp.java [hunk 1]
```diff
@@ -17,6 +17,17 @@
 
   private static final String ACTION_CHECKINTENT = "checkIntent";
 
+  private static String externalString;
+
+  // used to store external string from an external Java class method that
```
Comment (ddho23) [inline src/android/nl/xservices/plugins/LaunchMyApp.java]: minor: prefer /\* _/ or javadoc (/_\* **/) style comments when its more than 2 lines

**Comment by ddho23** [comment]
looks good after the formatting fix

**Comment by ddho23** [comment]
LGTM

**Comment by stevechae** [comment]
Merged to master. Closing.

**Pull request closed without merging**